kubernetes/data_source_kubernetes_endpoints_test.go

```diff
@@ -0,0 +1,65 @@
+package kubernetes
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	api "k8s.io/api/core/v1"
+)
+
+func TestAccKubernetesDataSourceEndpoints_basic(t *testing.T) {
+	var conf api.Endpoints
+	name := fmt.Sprintf("tf-acc-test-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		IDRefreshName:     "kubernetes_endpoints.test",
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckKubernetesEndpointDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccKubernetesEndpointsConfig_basic(name),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckKubernetesEndpointExists("kubernetes_endpoints.test", &conf),
+					resource.TestCheckResourceAttr("kubernetes_endpoints.test", "metadata.0.name", name),
```

Comment:
Could you please replace `kubernetes_endpoints.test` and `data.kubernetes_endpoints_v1.test` with variables? For example, as in this [test file](https://github.com/hashicorp/terraform-provider-kubernetes/blob/main/kubernetes/data_source_kubernetes_mutating_webhook_configuration_v1_test.go#L13)